src/frameworks/next/index.ts

```diff
@@ -660,9 +676,12 @@ export async function ɵcodegenFunctionsDirectory(
     await copy(join(sourceDir, "public"), join(destDir, "public"));
   }
 
-  // Add the `sharp` library if app is using image optimization
+  // Add the `sharp` library if app is using image optimization and is less than next@15
   if (await isUsingImageOptimization(sourceDir, distDir)) {
-    packageJson.dependencies["sharp"] = SHARP_VERSION;
+    const nextVersion = getNextVersion(sourceDir);
+    if (!nextVersion || !gte(nextVersion, "15.0.0")) {
```

Comment:
probably `lt` here makes more sense so we don't need to invert the logic?
```suggestion
    if (!nextVersion || lt(nextVersion, "15.0.0")) {
```

Reply:
Reverting this for now since images were erroring.